plugins/kubernetes/src/components/KubernetesContent/KubernetesContent.tsx

```diff
@@ -105,16 +107,24 @@ export const KubernetesContent = ({ entity }: KubernetesContentProps) => {
   >(undefined);
   const [error, setError] = useState<string | undefined>(undefined);
 
+  const googleAuth = useApi(googleAuthApiRef);
   useEffect(() => {
-    kubernetesApi
-      .getObjectsByServiceId(entity.metadata.name)
-      .then(result => {
-        setKubernetesObjects(result);
-      })
-      .catch(e => {
-        setError(e.message);
-      });
-  }, [entity.metadata.name, kubernetesApi]);
+    (async () => {
+      // TODO: Query Kubernetes backend to get info on which auth providers are in use
+      const googleAuthToken: string = await googleAuth.getAccessToken(
+        'https://www.googleapis.com/auth/cloud-platform',
+      );
+      const authTokens: AuthTokens = { google: googleAuthToken };
```

Comment:
We'll need error handling at some point here, and a place to call other auth providers.